src/Core/Stock/StockManager.php

```diff
@@ -317,9 +317,14 @@ protected function sendLowStockAlert($product, $id_product_attribute, $newQuanti
                 $templateVars['{lastname}'] = $employee->lastname;
 
                 Mail::Send(
-                    $idLang,
+                    $context->language->local,
```

Comment:
```suggestion
                    $context->language->id,
```

local property does not exist :thinking: